DocumentationValidator/Program.cs

```diff
@@ -32,15 +91,48 @@ static async Task<int> Main(string[] args)
                 }
             }
             Console.WriteLine("Completed checking log codes for documentation.");
+            return undocumentedLogCodes;
+        }
 
-            Console.WriteLine($"Undocumented Log Codes Count {undocumentedLogCodes.Count}");
+        private static async Task<Dictionary<NuGetLogCode, List<string>>> GetIssuesForLogCodeAsync(GitHubClient client, List<NuGetLogCode> logCodes)
+        {
+            Dictionary<NuGetLogCode, List<string>> issues = new(logCodes.Count);
 
-            foreach (var logCode in undocumentedLogCodes)
+            foreach (var logCode in logCodes)
             {
-                Console.Error.WriteLine(logCode);
+                issues.Add(logCode, new List<string>());
             }
+            IEnumerable<Issue> githubIssues = await GetAllIssues(client, "nuget", "docs.microsoft.com-nuget");
 
-            return undocumentedLogCodes.Any() ? 1 : 0;
+            foreach (Issue githubIssue in githubIssues)
+            {
+                foreach (var logCode in logCodes)
+                {
+                    if (IsIssueRelevantForLogCode(githubIssue, logCode))
+                    {
+                        issues[logCode].Add(githubIssue.HtmlUrl);
+                    }
+                }
+            }
+
+            return issues;
+        }
+
+        private static bool IsIssueRelevantForLogCode(Issue githubIssue, NuGetLogCode logCode)
+        {
+            return githubIssue.Title.Contains(logCode.ToString(), StringComparison.OrdinalIgnoreCase);
+        }
+
+        public static async Task<IEnumerable<Issue>> GetAllIssues(GitHubClient client, string org, string repo)
+        {
+            return await client.Issue.GetAllForRepository(
+                org,
+                repo,
+                new RepositoryIssueRequest
+                {
+                    Filter = IssueFilter.All
```

Comment:
I'm not familiar with GitHub's APIs. Does IssueFilter.All mean that it also returns closed issues?

Honestly, I haven't formed an opinion about whether or not it should search closed issues. I'm just curious.